chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup

```diff
@@ -160,9 +160,9 @@ fi
 yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/$COPR_OWNER/$COPR_PROJECT/repo/epel-7/$COPR_OWNER-$COPR_PROJECT-epel-7.repo
 yum -y install epel-release
 
-if ! $PROXY yum -y install \$CMIT python-mock; then
+if ! $PROXY yum -y install python-django \$CMIT python-mock; then
```

Comment:
It's interesting that this works given:

```
# repoquery -q --whatprovides python-django
python-django-0:1.6.11.6-1.el7.noarch
python-django16-0:1.6.11.6-6.el7.noarch
```

I would have expected _YUM_ to have chosen _python-django16-0:1.6.11.6-6_ with the higher _version-release_ given that they both satisfy the request of _python-django_.

Just as a data point, the other way of working around this would have been to add `--exclude=python-django16` to the `yum` command.